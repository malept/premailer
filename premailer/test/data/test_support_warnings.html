<html>
<head>
<title>Title</title>
<style type="text/css">
.text { margin:2px }
</style>
</head>
<body>
<h1 class="text">Hi!</h1>
<p><strong class="text">Yes!</strong></p>
</body>
</html>
