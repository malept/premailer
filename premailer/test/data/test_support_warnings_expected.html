<html>
<head>
<title>Title</title>
</head>
<body>
<h1 style="margin:2px">Hi!</h1>
<p><strong style="margin:2px">Yes!</strong></p>
</body>
</html>
